doc/source/data/examples/nyc_taxi_basic_processing.ipynb

```diff
@@ -316,7 +316,7 @@
       "id": "a3fb551b",
       "metadata": {},
       "source": [
-       "To get a better sense of the data size, we can calculate the size in bytes of the full dataset. Note that for Parquet files, this size-in-bytes will be pulled from the Parquet metadata (not triggering a data read) and will therefore be the on-disk size of the data; this might be significantly smaller than the in-memory size!"
+       "To get a better sense of the data size, we can calculate the size in bytes of the full dataset. Note that for Parquet files, this size-in-bytes will be pulled from the Parquet metadata (not triggering a data read), and will therefore might be significantly different than the in-memory size!"
```

Comment:
maybe add an explanation about compression and sometimes non-deterministic compression ratio ?

Reply:
Do you mean the `size_bytes()` calls to be non-deterministic? It's actually deterministic, but just inaccurate compared to in-memory size.